src/components/UnstyledLink/UnstyledLink.tsx

```diff
@@ -16,6 +16,16 @@ export const UnstyledLink = React.memo(
     props,
     _ref,
   ) {
+    const linkRef = useRef<HTMLAnchorElement>(null);
+
+    useImperativeHandle(_ref, () => ({
```

Comment:
Since we're using `_ref` now we can remove the underscore as it denotes unused variables.